components/Notifications/Notification.tsx

```diff
@@ -0,0 +1,113 @@
+import { LoadingSpinner } from "components/LoadingSpinner";
+import NextLink from "next/link";
+import Check from "public/assets/icons/check.svg";
+import Close from "public/assets/icons/close.svg";
+import Warning from "public/assets/icons/warning.svg";
+import styled from "styled-components";
+import { NotificationT } from "types";
+
+export function Notification({
```

Comment:
This is a presentational component that shows a contract interaction notification. It handles three types: "success" | "error" | "pending"